### PR TITLE
use consent feature flag

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

https://ticketsystem.dbildungscloud.de/browse/BC-2289

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/pages/administration/StudentOverview.page.vue [hunk 1]
```diff
@@ -398,10 +398,7 @@ export default {
 			};
 		},
 		isConsentNecessary() {
-			if (process.env.SC_THEME === "brb") {
-				return false;
-			}
-			return true;
+			return this.env.FEATURE_CONSENT_NECESSARY;
```
Comment (virgilchiriac) [inline src/pages/administration/StudentOverview.page.vue:401]: why calling it with this.env, while above envConfigModule
in some other cases, I see being called like envConfigModule.getEnv.FEATURE_CONSENT_NECESSARY

Comment (odalys-dataport) [reply src/pages/administration/StudentOverview.page.vue:401]: Since envConfigModule.getEnv is already stored in "env" I just went with the flow that was already in place. I can change it though.

**Review by MartinSchuhmacher — DISMISSED** [review]
tests missing in src/store/env-config.unit.ts
also think about test in src/pages/administration/student-overview.unit.js

File: src/store/env-config.ts [hunk 2]
```diff
@@ -42,6 +42,7 @@ export default class EnvConfigModule extends VuexModule {
 		SC_TITLE: "",
 		SC_SHORT_TITLE: "",
 		GHOST_BASE_URL: "",
+		FEATURE_CONSENT_NECESSARY: true,
```
Comment (virgilchiriac) [inline src/store/env-config.ts:45]: the logic in a bit hard to follow. So here we set a default in nuxt, which is then overwritten by the value we get from server, right?

Comment (odalys-dataport) [reply src/store/env-config.ts:45]: This confused me as well, this is due to typescript complaining about it not being declared

**Review by virgilchiriac — APPROVED** [review]
